### PR TITLE
[one-cmds] Install model2nnpkg

### PR DESCRIPTION
**Pull request by hseok-oh**
Let's install model2nnpkg script in one-cmds cmake.

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

---

Draft: #10235

### CODE REVIEW

**Review by mhs4670go — APPROVED** [review]
LGTM

File: compiler/one-cmds/CMakeLists.txt [hunk 1]
```diff
@@ -41,7 +41,7 @@ foreach(ONE_COMMAND IN ITEMS ${ONE_COMMAND_FILES})
                       GROUP_READ GROUP_EXECUTE
                       WORLD_READ WORLD_EXECUTE
           DESTINATION bin)
-          
+
```
Comment (seanshpark) [inline compiler/one-cmds/CMakeLists.txt:44]: please remove this change. better to put in another PR.

Comment (hseok-oh) [reply compiler/one-cmds/CMakeLists.txt:44]: Ah. This diff is by vscode's setting. I'll fix it manually.

File: compiler/one-cmds/CMakeLists.txt [hunk 2]
```diff
@@ -74,6 +74,15 @@ foreach(ONE_UTILITY IN ITEMS ${ONE_UTILITY_FILES})
...
 endforeach(ONE_UTILITY)
 
+# one-pack internally uses model2nnpkg tool
+set(MODEL2NNPKG "${NNAS_PROJECT_SOURCE_DIR}/tools/nnpackage_tool/model2nnpkg/model2nnpkg.sh")
+install(FILES ${MODEL2NNPKG}
+              PERMISSIONS OWNER_WRITE OWNER_READ OWNER_EXECUTE
+                          GROUP_READ GROUP_EXECUTE
+                          WORLD_READ WORLD_EXECUTE
+              DESTINATION bin
+              RENAME "model2nnpkg")
```
Comment (seanshpark) [inline compiler/one-cmds/CMakeLists.txt:84]: FYI there are other scripts that use as `model2nnpkg.sh`, and I assume you will change them :)

**Review by seanshpark — APPROVED** [review]
LGTM

File: compiler/one-cmds/CMakeLists.txt (hunk 2 above)
Comment (hseok-oh) [reply compiler/one-cmds/CMakeLists.txt:84]: Right.

**Review by mhs4670go — APPROVED** [review]
